packages/web/src/scss/components/RadioField/_theme.scss

```diff
@@ -1,22 +1,15 @@
 @use '@tokens' as tokens;
+@use '../../theme/form-fields' as form-fields-theme;
 
 $spacing: tokens.$space-400;
 
 $input-border-radius: 50%;
-$input-outline-width: 2px;
 $input-width: 20px;
 $input-inner-dot-width: 10px;
 $input-inner-dot-width: 10px;
```

Comment:
Maybe in `rem`/`em`?

Reply:
I would choose `rem`, but `em` will make a sense with more size variants...